src/components/ControlBar/ShowMoreButton.js

```diff
@@ -1,30 +1,43 @@
 import React from 'react';
 import i18n from '@dhis2/d2-i18n';
+import { colors } from '@dhis2/ui-core';
 import { withStyles } from '@material-ui/core/styles';
 
-import { colors } from '../../modules/colors';
-
 export const SHOWMORE_BAR_HEIGHT = 16;
 
 const styles = {
     showMore: {
-        color: colors.royalBlue,
+        color: colors.grey700,
         cursor: 'pointer',
         fontSize: 11,
-        fontWeight: 700,
+        fontWeight: 400,
         height: SHOWMORE_BAR_HEIGHT,
         paddingTop: 5,
-        textTransform: 'uppercase',
         visibility: 'visible',
```

Comment:
Are the visibility and height properties still necessary? i see they are not applied when the button has class "disabled".

Reply:
Thanks for reporting, unused styles have been removed.